app/models/solr_document.rb

```diff
@@ -149,4 +149,64 @@ def degree_name
   def graduation_year
     self[Solrizer.solr_name('graduation_year')]
   end
+
```

Comment:
Yeah, I did something like that in the tests that I just pushed up. Id definitely be willing to simplify this a bit. 👍